src/libs/error.js

```diff
@@ -1,30 +1,12 @@
-import _ from 'lodash/fp'
+import { notify, sessionTimeoutProps } from 'src/components/Notifications'
 import { reloadAuthToken, signOut } from 'src/libs/auth'
-import * as StateHistory from 'src/libs/state-history'
-import * as Utils from 'src/libs/utils'
 
 
-export const errorStore = Utils.atom([])
-
-const addError = item => errorStore.update(state => _.concat(state, [item]))
-
 export const reportError = async (title, obj) => {
   if (obj instanceof Response && obj.status === 401 && !(await reloadAuthToken())) {
-    addError({ title: 'Session timed out', error: 'You have been signed out due to inactivity', code: 'sessionTimeout' })
+    notify('Session timed out', sessionTimeoutProps)
     signOut()
   } else {
-    addError({ title, error: await (obj instanceof Response ? obj.text() : obj) })
+    notify('error', title, { detail: await (obj instanceof Response ? obj.text() : obj) })
```

Comment:
This will no longer automatically stack errors together (unless they happen to have the same title), right? Just checking to make sure that's the behavior we want.

Reply:
Correct. I talked with Lou about this... and I'm pretty sure I talked about it with Brad as well. This makes it much easier to identify if 1 error is different among a group of 10 other errors that are all the same.